org-roam-protocol.el

```diff
@@ -58,6 +63,20 @@ It opens or creates a note with the given ref.
       (error "No ref key provided"))
     (when-let ((title (cdr (assoc 'title decoded-alist))))
       (push (cons 'slug (funcall org-roam-title-to-slug-function title)) decoded-alist))
+    (let-alist decoded-alist
```

Comment:
let-alist seems to be a package on elpa, not sure if it ships with Emacs by default.

Also from the docs of `let-alist`, it seems the backslashes aren't necessary.

Reply:
`let-alist` should be in core, but we might require `let-alist` for it to work.

The backslashes aren't necessary, but `lispy` is adding them for me.  I think that's a problem on their end, so I'll edit by hand and file a report there.